code/gits.py

```diff
@@ -180,5 +172,21 @@
 gits_stash_subparser = subparsers.add_parser("stash")
 gits_stash_subparser.set_defaults(func=gits_stash)
 
+gits_commit_subparser = subparsers.add_parser('commit')
+gits_commit_subparser.add_argument('-m',
+                                   required=True,
+                                   help='git commit message')
+
+
+gits_squash_subparser = subparsers.add_parser('squash')
+gits_squash_subparser.add_argument('-n',
+                                   required=True,
```

Comment:
can we try using the gits_add and gits_commit functions to implement this functionality instead of repeating "add" and "commit" subprocesses here?